applications/app/views/videoEmbed.scala.html

```diff
@@ -1,4 +1,4 @@
-@(page: EmbedPage)(implicit request: RequestHeader)
+@(page: model.EmbedPage)(implicit request: RequestHeader)
```

Comment:
moving `EmbedPage` into `meta`? Following the current pattern of objects that extend `Page` living there e.g
 https://github.com/guardian/frontend/blob/master/common/app/model/meta.scala#L437